apps/emqx/src/emqx_listeners.erl

```diff
@@ -388,7 +388,11 @@ do_start_listener(quic, ListenerName, #{bind := Bind} = Opts) ->
                 ] ++
                     case maps:get(cacertfile, SSLOpts, undefined) of
                         undefined -> [];
-                        CaCertFile -> [{cacertfile, binary_to_list(CaCertFile)}]
+                        CaCertFile -> [{cacertfile, str(CaCertFile)}]
+                    end ++
+                    case maps:get(password, SSLOpts, undefined) of
+                        undefined -> [];
+                        Password -> [{password, str(Password)}]
```

Comment:
Should we wrap this password here with `emqx_secret:wrap`?  Or esockd would still need to support it, I guess?

Reply:
I guess no, because this option is passed down to quic / ssl lib

Reply:
I could add support in quicer, but I don't know from where we should start to wrap it? from emqx_schema ?